tests/lib/schema_validation_test.py

```diff
@@ -151,6 +153,78 @@ def test_valid_without_auto_neg_and_missing_speed(self, default_data):
         libnmstate.validator.validate(default_data)
 
 
+class TestIfaceTypeVxlan(object):
+    def test_invalid_vxlan_without_options(self, default_data):
+        default_data[INTERFACES].append(
+            {
+                Interface.NAME: 'vxlan0',
+                Interface.TYPE: VXLAN.TYPE,
+                VXLAN.CONFIG_SUBTREE: {},
+            }
+        )
+
+        with pytest.raises(js.ValidationError) as err:
+            libnmstate.validator.validate(default_data)
+
+        assert "'vxlan' is not one of ['unknown']" in err.value.args[0]
```

Comment:
this error msg is a bit cryptic isn't it ? I'm surprised at least.

Reply:
This is how the error at jsonschema works, since there is no schema that matches it it mark it as unknown, it does not show you the partial errors or the like.